scripts/features/elasticsearch.sh

```diff
@@ -38,11 +38,10 @@ echo "Elasticsearch majorVersion: $majorVersion"
 
 
 # Install Java & Elasticsearch
-
-wget -qO - https://artifacts.elastic.co/GPG-KEY-elasticsearch | sudo apt-key add -
+curl -fsSL https://artifacts.elastic.co/GPG-KEY-elasticsearch | sudo gpg --dearmor -o /etc/apt/keyrings/elasticsearch.gpg
 
 if [ ! -f /etc/apt/sources.list.d/elastic-$majorVersion.x.list ]; then
-    echo "deb https://artifacts.elastic.co/packages/$majorVersion.x/apt stable main" | sudo tee -a /etc/apt/sources.list.d/elastic-$majorVersion.x.list
+    echo "deb [signed-by=/etc/apt/keyrings/elasticsearch.gpg] https://artifacts.elastic.co/packages/$majorVersion.x/apt stable main" | sudo tee -a /etc/apt/sources.list.d/elastic-$majorVersion.x.list
```

Comment:
do we *want* to append to the source file here? Since it looks like we're already creating separate files for each version, and checking if the file already doesn't exist, I think the `-a` is superfluous.

Reply:
Updated to drop `-a`